app/src/main/res/layout/fragment_pokemon_type_layout.xml

```diff
@@ -1,15 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <TextView
+    <ProgressBar
+        android:id="@+id/pokemon_type_progress_bar"
+        style="?android:attr/progressBarStyle"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_marginTop="@dimen/fragment_text_view_margin_top"
-        android:fontFamily="@font/pokemon_font"
-        android:text="@string/pokemon_type_fragment"
-        android:textAlignment="center"
-        android:textSize="@dimen/fragment_text_view_text_size" />
+        android:layout_height="wrap_content"
+        android:visibility="gone"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/pokemon_type_recycler_view"
+        android:layout_width="wrap_content"
+        android:layout_height="0dp"
```

Comment:
this can be replaced by match_parent and remove the bottom and top constraints

Reply:
fixed, thanks